File: ExtraCredit/ThirdLightbulb/Fibonacci.py
def nthFibonacci(n):
    if n == 0:
        return 0
    a, b = 0, 1
    for i in range(2, n+1):
        c = a + b
        a = b
        b = c
    return b
print(nthFibonacci(8))

File: ExtraCredit/ThirdLightbulb/Fibonacci.java
package ExtraCredit.ThirdLightbulb;

public class Fibonacci {
    public static int nthFibonacci(int n) {
        if (n == 0) return 0;
        int a = 0, b = 1;
        for (int i = 2; i <= n; i++) {
            int c = a + b;
            a = b;
            b = c;
        }
        return b;
    }

    public static void main(String[] args) {
        if (args.length > 0)
            System.out.println(nthFibonacci(Integer.parseInt(args[0])));
    }
}
